### ISSUE TITLE
[Bug] useRowSelect and useTable#getSubRows does not work as expected

### ISSUE THREAD

**Issue by xiaoyu-tamu** [tannerlinsley/react-table#2609]
**Describe the bug (required)**

TLDR; When `getRows` defined, subRows are not get selected when parent selected.

Looking into the src code, https://github.com/tannerlinsley/react-table/blob/master/src/plugin-hooks/useRowSelect.js#L170
`useSelected` retrieve subRows using `getSubRow()`, it pass the `row instance` instead of `row original data`. Which cause the problem.

I think we are safe to use `row.subRows` here since subRows already computed inside useTable https://github.com/tannerlinsley/react-table/blob/master/src/hooks/useTable.js#L549-L558

![image](https://user-images.githubusercontent.com/33362998/89718029-9f02f080-d981-11ea-80b8-d9a2bc9f1fc2.png)

**Provide an example via Codesandbox! (required)**
https://codesandbox.io/s/priceless-currying-5mwss?file=/src/App.js

**Steps To Reproduce (required)**

1. Go to 
2. Click on first row
3. Click on expand

**Expected behavior (Recommended)**
All sub rows should get selected

**Additional context**

**Comment by tannerlinsley** [comment]
This should be fixed in the latest version.

**Comment by xiaoyu-tamu** [comment]
@tannerlinsley  I also tried this one for v7.5.2, the problem still exist ?

![image](https://user-images.githubusercontent.com/33362998/95038657-4e172c00-0694-11eb-881d-e6cc5da1ca9b.png)

**Comment by aaryn101** [comment]
@tannerlinsley this is definitely still a bug and should be re-opened. You can confirm it via the original codesandbox link (with the latest react-table version).

**Comment by xiaoyu-tamu** [comment]
There is a PR maybe related this problem #2886, looks like @tannerlinsley working on the next version and not accept any PR for v7. :-(
